homeassistant/components/amberelectric/config_flow.py

```diff
@@ -23,20 +23,24 @@
 
 def generate_site_selector_name(site: Site) -> str:
     """Generate the name to show in the site drop down in the configuration flow."""
+    # For some reason the generated API key returns this as any, not a string. Thanks pydantic
+    nmi = str(site.nmi)
     if site.status == SiteStatus.CLOSED:
-        return site.nmi + " (Closed: " + site.closed_on.isoformat() + ")"  # type: ignore[no-any-return]
-    if site.status == SiteStatus.PENDING:
-        return site.nmi + " (Pending)"  # type: ignore[no-any-return]
-    return site.nmi  # type: ignore[no-any-return]
+        if site.closed_on is None:
+            return f"{nmi} (Closed)"
+        return f"{nmi} (Closed: {site.closed_on.isoformat()})"
+    if site.status == "pending":
+        return f"{nmi} (Pending)"
+    return nmi
 
 
 def filter_sites(sites: list[Site]) -> list[Site]:
     """Deduplicates the list of sites."""
     filtered: list[Site] = []
     filtered_nmi: set[str] = set()
 
-    for site in sorted(sites, key=lambda site: site.status.value):
-        if site.status == SiteStatus.ACTIVE or site.nmi not in filtered_nmi:
+    for site in sorted(sites, key=lambda site: site.status):
+        if site.status == "active" or site.nmi not in filtered_nmi:
```

Comment:
or did the site statuses got removed

Reply:
The new pip module doesn't use the constants. It's auto generated, and doesn't expose enum's anymore

Reply:
That enum is still present here:

https://github.com/madpilot/amberelectric.py/blob/011988cb0d9cbcccb3ff714ac2cb5b7a7a5b511e/amberelectric/models/site_status.py#L30

Reply:
Good pickup. Fixed